lib/state_abbr.rb

```diff
@@ -5,19 +5,19 @@ def get_uppercase_symbol(abbr)
 
 # Returns the state name associated with the state abbreviation abbr.
 # abbr: A two character (uppercase or lowercase) symbol or string.
-def get_state_name_from_abbr(abbr)
+def get_us_state_from_abbr(abbr)
   return STATE_ABBR_HASH[get_uppercase_symbol(abbr)]
 end
 
 # Returns the state name associated with the state (including Washington DC)
 # abbreviation abbr.
 # abbr: A two character (uppercase or lowercase) symbol or string.
-def get_state_name_with_dc_from_abbr(abbr)
+def get_us_state_with_dc_from_abbr(abbr)
   abbr = get_uppercase_symbol(abbr)
   if abbr == :DC
     return 'Washington DC'
```

Comment:
Can we add `DC: 'Washington DC'` to `STATE_ABBR_HASH` to get rid of this special case (and this method)?